File: modern/templates/java/io/github/notenoughupdates/moulconfig/test/InnerCategory.java
package io.github.notenoughupdates.moulconfig.test;

import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;

public class InnerCategory {
    @ConfigEditorBoolean
    @ConfigOption(name = "Test Option", desc = "Test toggle")
    public boolean shouldTestToggle = false;
}

File: modern/templates/java/io/github/notenoughupdates/moulconfig/test/InnerCategory.kt
package io.github.notenoughupdates.moulconfig.test

import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean
import io.github.notenoughupdates.moulconfig.annotations.ConfigOption

class InnerCategory {

    @ConfigEditorBoolean
    @ConfigOption(name = "Test Option", desc = "Test toggle")
    var shouldTestToggle: Boolean = false

}
